runtime_config_linux.go

```diff
@@ -216,6 +222,8 @@ type Resources struct {
 	HugepageLimits []HugepageLimit `json:"hugepageLimits,omitempty"`
 	// Network restriction configuration
 	Network *Network `json:"network,omitempty"`
+	// IntelRdt restriction configuration
+	IntelRdt IntelRdt `json:"intelRdt,omitempty"`
```

Comment:
Thanks. Changed the types of IntelRdt and L3Cbm to **pointer** according to #233.
Commit: 233311be23d559919fbe3b9890de7319ce115d54